shuffler/auxiliary/NW Sea/Isle of Gust/Gust.json

```
@@ -0,0 +1,114 @@
+{
+  "$schema": "../../../aux_schema.json",
+  "name": "Gust",
+  "rooms": [
+    {
+      "name": "LowerIsland",
+      "chests": [
+        {
+          "name": "DigSpot1",
+          "contents": "small_red_rupee",
+          "type": "dig_spot",
+          "zmb_file_path": "TODO",
+          "zmb_actor_index": 0
+        },
+        {
+          "name": "DigSpot2",
+          "contents": "small_red_rupee",
+          "type": "dig_spot",
+          "zmb_file_path": "TODO",
+          "zmb_actor_index": 0
+        },
+        {
+          "name": "DigSpot3",
+          "contents": "small_red_rupee",
+          "type": "dig_spot",
+          "zmb_file_path": "TODO",
+          "zmb_actor_index": 0
+        },
```

See above comment about missing dig spot in `Gust.logic`
```suggestion
        },
        {
          "name": "DigSpot4",
          "contents": "big_green_rupee",
          "type": "dig_spot",
          "zmb_file_path": "TODO",
          "zmb_actor_index": 0
        },
```